aep/general/0008/aep.md.j2

```diff
@@ -215,6 +215,9 @@ API design examples in AEPs **should** be presented in both [OpenAPI][] and
 the concept. When using RPCs in examples, a `google.api.http` annotation
 **should** be included.
 
+[OpenAPI][] examples **must** be authored as
+[OAS version 3.1](https://spec.openapis.org/oas/v3.1.1.html).
```

Comment:
because the link is to 3.1.1, consider the text being 3.1.x?

Reply:
sure, that works.

Reply:
done! PTAL.